src/libslic3r/PerimeterGenerator.cpp

```diff
@@ -1005,17 +1051,17 @@ static ExtrusionEntityCollection traverse_extrusions(const PerimeterGenerator& p
                     // Fuzzy splitted extrusion
                     if (std::all_of(splitted.begin(), splitted.end(), [](const Algorithm::SplitLineJunction& j) { return j.clipped; })) {
                         // The entire polygon is fuzzified
-                        fuzzy_extrusion_line(extrusion.junctions, r.first);
+                        fuzzy_extrusion_line(extrusion.junctions, slice_z, r.first);
                     } else {
                         const auto current_ext = extrusion.junctions;
                         std::vector<Arachne::ExtrusionJunction> segment;
                         segment.reserve(current_ext.size());
                         extrusion.junctions.clear();
 
-                        const auto fuzzy_current_segment = [&segment, &extrusion, &r]() {
+                        const auto fuzzy_current_segment = [&segment, extrusion, &r, slice_z]() {
```

Comment:
I'm getting a compile error here: because `extrusion` should IMHO be passed by reference like before.